spec/requests/users_controller_spec.rb

```diff
@@ -5887,7 +5887,8 @@ def create_totp
             expect(response.status).to eq(200)
 
             expect(user1.reload.user_second_factors).to be_empty
-            expect(user1.security_keys).to be_empty
+            expect(user1.second_factor_security_keys).to be_empty
+            expect(user1.security_keys.length).to eq(1)
```

Comment:
I think it will be better to be more specific with our assertion here and assert for the actual objects instead of the count since an incorrect object will still lead to a count of `1`.